qa/rpc-tests/test_framework/authproxy.py

```diff
@@ -67,7 +67,18 @@ def EncodeDecimal(o):
 class AuthServiceProxy(object):
```

Comment:
Instead of changing autoproxy, I'd prefer if this was an object that wrapped an authproxy, and logs and passes on all the calls. I think it could be done that way